src/Conn.h

```diff
@@ -60,7 +60,7 @@ struct ConnTuple {
     uint32_t src_port = 0;
     uint32_t dst_port = 0;
     bool is_one_way = false; // if true, don't canonicalize order
-    TransportProto proto = TRANSPORT_UNKNOWN;
+    uint8_t proto;
```

Comment:
`uint8_t` makes sense from the IP header's perspective, but my `/etc/protocols` shows 262 for multipath TCP, though with the caveat that it's only used internally by the kernel. It likely doesn't matter for our use case, and we could broaden later as needed, but I wanted to flag it.

Reply:
Good point, bumped it up to a uin16_t and set the "unknown" value to 65535.